src/SignalR/common/Http.Connections/src/Internal/HttpConnectionManager.cs

```diff
@@ -176,7 +165,7 @@ public void Scan()
                 }
                 else
                 {
-                    if (!Debugger.IsAttached && _useSendTimeout)
```

Comment:
I see that there's still `var utcNow = DateTimeOffset.UtcNow;` above this in the Scan method(). Why isn't this using the ISystemClock?

Reply:
Because I scoped the change back to only be at the SignalR layer

Reply:
Just the Hub layer? Why?

Reply:
Because the reason for the clock is mainly for tests right now, and the layering between the HttpConnection layer and the Hub layer makes it difficult/not possible to share a clock without designing a public feature.

Reply:
We can always look at plumbing it all the way later if we need to.